pkgs/development/python-modules/flexmock/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, nose
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "flexmock";
+  version = "0.10.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fe95c8727f4db73dc8f2f7b4548bffe7992440a965fefd60da291abda5352c2b";
+  };
+
+  checkInputs = [ nose pytest ];
+
+  meta = with lib; {
+    homepage = https://flexmock.readthedocs.io/en/latest/;
+    description = "flexmock is a testing library for Python.";
```

Comment:
`description` should not have the name of said software in it or a period at the end.
> https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does. 

```suggestion
    description = "A testing library for Python that makes it easy to create mocks, stubs, and fakes";
```